moto/ec2/models.py

```diff
@@ -6089,6 +6090,50 @@ def get_all_transit_gateway_route_tables(self, filters):
     def delete_transit_gateway_route_table(self, transit_gateway_route_table_id):
         return self.transit_gateways_route_tables.pop(transit_gateway_route_table_id)
 
+    def create_transit_gateway_route(
+        self,
+        transit_gateway_route_table_id,
+        destination_cidr_block,
+        transit_gateway_attachment_id=None,
+        blackhole=False
+    ):
+        transit_gateways_route_table = self.transit_gateways_route_tables[transit_gateway_route_table_id]
+        transit_gateways_route_table.routes[destination_cidr_block] = {
+            "destinationCidrBlock": destination_cidr_block,
+            "prefixListId": "random()",
+            "state": "blackhole" if blackhole else "active",
+            "transitGatewayAttachments": {
+                "resourceId": "String",
```

Comment:
Should we add actual values for `resourceId` and `resourceType` here - looks like these values are only placeholders / schema types?

Reply:
We need transitGatewayAttachments to achieve this. We will make it dynamic once we implement the TransitGatewayAttachment.

Reply:
Sounds good - let's please add a `TODO` comment in the code there and indicate in the attribute responses that this is so far still mocked:
```
            "transitGatewayAttachments": {
                # TODO: needs to be fixed once we have support for transit gateway attachments
                "resourceId": "TODO",
                "resourceType": "TODO",
```